test/e2e/standard/cluster.go

```diff
@@ -413,6 +413,65 @@ func (sc *SanityChecker) checkCanUseAzureFileStorage(ctx context.Context) error
 	return nil
 }
 
+func (sc *SanityChecker) checkEnforcesEmptyDirQuotas(ctx context.Context) error {
+	namespace, err := sc.createProject(ctx)
+	if err != nil {
+		return err
+	}
+	defer sc.deleteProject(ctx, namespace)
+
+	// create test pod
+	podName := "busybox"
+	By("Creating busybox pod to test EmptyDir Quotas")
+	_, err = sc.Client.EndUser.CoreV1.Pods(namespace).Create(&corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: podName,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:  podName,
+					Image: podName,
+					Command: []string{
+						"/bin/dd",
+						"if=/dev/urandom",
+						fmt.Sprintf("of=/cache/%s.bin", namespace),
+						"bs=1M",
+						"count=2000",
+					},
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "cache-volume",
+							MountPath: "/cache",
+						},
+					},
+				},
+			},
+			Volumes: []corev1.Volume{
+				{
+					Name: "cache-volume",
+					VolumeSource: v1.VolumeSource{
+						EmptyDir: &corev1.EmptyDirVolumeSource{},
+					},
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyNever,
+		},
+	})
+	if err != nil {
+		return err
+	}
+	By("Created pod")
+	By(fmt.Sprintf("Waiting for pod %s to finish", podName))
+	err = wait.PollImmediate(2*time.Second, 10*time.Minute, ready.CheckPodHasPhase(sc.Client.Admin.CoreV1.Pods(namespace), podName, corev1.PodFailed))
```

Comment:
you created the pod with the Enduser client so you probably should use the same client in CheckPodHasPhase
```suggestion
	err = wait.PollImmediate(2*time.Second, 10*time.Minute, ready.CheckPodHasPhase(sc.Client.Enduser.CoreV1.Pods(namespace), podName, corev1.PodFailed))
```

Reply:
make sense, copy/paste syndrom